pages/rental.js

```diff
@@ -0,0 +1,198 @@
+import React,{Component} from 'react';

```

Comment:
This should be there in ```components/page/Rental/index.js````

Reply:
have moved as above in the latest commit but not able to run the code